hicexplorer/HiCMatrix.py

```diff
@@ -381,8 +388,10 @@ def getRegionBinRange(self, chrname, startpos, endpos):
             exit()
 
         try:
-            startbin = sorted(self.interval_trees[chrname][startpos:startpos + 1])[0].data
-            endbin = sorted(self.interval_trees[chrname][endpos:endpos + 1])[0].data
+            startbin = sorted(
+                self.interval_trees[chrname][startpos:startpos + 1])[0].data
```

Comment:
please revert these changes, it makes everything more complicated and its hard to review.